### PR TITLE
feat: change the metrics to dashboard

### LINKED ISSUE #1627: improve Metrics module
Here are some minor issues on the Login page we could resolve.

## TODO

- [ ] Metrics page name change to: Dashboard.
- [ ] Metrics page hint: Use browser localStorage mechanism to store Grafana links locally, as a preview page only, currently no substantive function.

### PR DESCRIPTION
**Pull request by stu01509**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

- Related issues
Fix #1627

### New feature or improvement
- Describe the details and related test reports.

___
### Please add the corresponding test cases if necessary.
Change the `metrics` to `dashboard`, and adjust the dashboard tip content.

image

image

### CODE REVIEW

File: web/config/routes.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ const routes = [
     component: './Metrics',
   },
   {
-    path: '/metrics',
+    path: '/dashboard',
     component: './Metrics',
   },
   {
```
Comment (stu01509) [inline web/config/routes.ts:26]: Hi @juzhiyuan 

Should I rename the `Metrics` component to `Dashboard`?

File: web/cypress/integration/metrics/metrics-smoketest.spec.js [hunk 2]
```diff
@@ -25,11 +25,11 @@ context('metrics page smoke test', () => {
...
-    cy.url().should('contains', '/metrics');
+    cy.contains('Dashboard').click();
+    cy.url().should('contains', '/dashboard');
     cy.get(this.domSelector.pageContent)
       .children()
-      .should('contain', 'Metrics')
+      .should('contain', 'Dashboard')
       .and('contain', 'You have not configured Grafana')
       .and('contain', 'Configure');
   });
```
Comment (stu01509) [inline web/cypress/integration/metrics/metrics-smoketest.spec.js:35]: Also, should I rename the test case folder and file name?

File: web/config/routes.ts (hunk 1 above)
Comment (juzhiyuan) [reply web/config/routes.ts:26]: Would better

File: web/src/pages/Metrics/locales/en-US.ts [hunk 3]
```diff
@@ -17,5 +17,5 @@
 export default {
   'page.metrics.empty.description.grafanaNotConfig': 'You have not configured Grafana',
   'page.metrics.button.grafanaConfig': 'Configure',
-  'page.metrics.tip': 'Use browser iframe to store monitor page URL, local only.',
+  'page.metrics.tip': 'Use browser localStorage to store monitor page URL, local only.',
 };
```
Comment (stu01509) [inline web/src/pages/Metrics/locales/en-US.ts:21]: Should I change the `'page.metrics.xxx` to `page.dashboard.xxx` and rename the folder?

File: web/cypress/integration/metrics/metrics-smoketest.spec.js (hunk 2 above)
Comment (juzhiyuan) [reply web/cypress/integration/metrics/metrics-smoketest.spec.js:35]: Would better but not needed

File: web/config/routes.ts (hunk 1 above)
Comment (stu01509) [reply web/config/routes.ts:26]: OK, I will improve it :)

File: web/src/pages/Metrics/locales/en-US.ts (hunk 3 above)
Comment (juzhiyuan) [reply web/src/pages/Metrics/locales/en-US.ts]: would better to use `metrics` all the paces

File: web/src/locales/zh-CN/menu.ts [hunk 4]
```diff
@@ -22,7 +22,7 @@ export default {
   'menu.login': '登录',
   'menu.register': '注册',
   'menu.register.result': '注册结果',
-  'menu.dashboard': 'Dashboard',
+  'menu.dashboard': '仪表板',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/menu.ts:25]:
```suggestion
  'menu.dashboard': 'Dashboard',
```

**Review by juzhiyuan — APPROVED** [review]

File: web/src/locales/zh-CN/menu.ts (hunk 4 above)
Comment (liuxiran) [reply web/src/locales/zh-CN/menu.ts]: may I ask why we keep `Dashboard`  in zh-CN? @juzhiyuan

**Review by nic-chen — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Pull request merged**